tests/test_provider.py

```diff
@@ -16,6 +16,12 @@ def test_report_info_od(provider):
         assert pinfo2['login'] == "personal-drive-owner"
 
 
+def test_globalize_oid(provider):
```

Comment:
Pls also add a test to globalize an oid that is not root

Reply:
good point, will do.